test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -509,7 +508,8 @@ def __init__(self, out_dir, diag_path=None, server_host=None,
                          ATTR_u, ATTR_SvrHost, ATTR_aclgroup, ATTR_acluser,
                          ATTR_aclResvgroup, ATTR_aclResvuser, ATTR_SchedHost,
                          ATTR_aclResvhost, ATTR_aclhost, ATTR_owner,
-                         ATTR_exechost, ATTR_NODE_Host, ATTR_NODE_Mom]
+                         ATTR_exechost, ATTR_NODE_Host, ATTR_NODE_Mom,
+                         ATTR_rescavail + ".host"]
```

Comment:
The requirement was to only obfuscate hostnames, so I didn't include the vnode names. Do you think we should also do the vnode names?